### PR TITLE
Update de_DE.lang

### PR DESCRIPTION
**Pull request by noname42**
fixed a lot of typos

### CODE REVIEW

File: de_DE.lang [hunk 1]
```diff
@@ -9,7 +9,7 @@ enchantment.special.sapgreat=Starkes Zehren
 potion.fluxtaint=Fluxvergiftung
 potion.soulshatter=Seelensplittern
 potion.visexhaust=Fluxgrippe
-potion.infvisexhaust=Fluxphage
+potion.infvisexhaust=Fluxvirus
```
Comment (Vexatos) [inline de_DE.lang]: This is not a type, "Phage" actually is a scientific term. But I'm fine with "Virus".

File: de_DE.lang [hunk 2]
```diff
@@ -19,16 +19,16 @@ potion.thaumarhia=Thaumarhoe
...
-warp.text.3=Selsame Stimmen enthüllen dir Geheimnisse.
+warp.text.3=Seltsame Stimmen enthüllen dir Geheimnisse.
 warp.text.4=Deine Sicht wird seltsam und schummrig.
 warp.text.5=Das Licht wird auf einmal unglaublich hell und verbrennt deine Haut.
 warp.text.6=Ein dichter Nebel kommt aus dem Nichts. Irgendetwas rührt sich in seinen Tiefen.
 warp.text.7=Sie sind überall! Lauf!
-warp.text.8=Es muss doch irgendeinen Weg geben, diese Kopfschmerzen zu loszuwerden... Oder?
-warp.text.9=Du fühlst dich plötzlich schlecht dabei, Dinge abzubauen.
+warp.text.8=Es muss doch irgendeinen Weg geben, diese Kopfschmerzen loszuwerden... Oder?
+warp.text.9=Du fühlst dich plötzlich zu schwach um Dinge abzubauen.
```
Comment (Vexatos) [inline de_DE.lang]: has to be `zu schwach, um`

File: de_DE.lang [hunk 3]
```diff
@@ -870,35 +870,35 @@ tc.research_category.GOLEMANCY=Golemantik
 tc.research_category.ELDRITCH=Unnatürliches
 #
 # Special formatting codes:
-# <BR> or <BR/>         Paragraph break (<BR/> is included for people using XML for their localization)
-# <LINE> or <LINE/>     Insert a fancy linebreak between sections of text.
-# <IMG>...</IMG>  Insert a centered image into text. Parameters are separated by ':' and are:
-#                    - mod resource location name
```
Comment (Vexatos) [inline de_DE.lang]: These all need to go back. They are in en_US.lang, so please revert every change done between line 873 and line 932.

File: de_DE.lang [hunk 4]
```diff
@@ -1125,11 +1125,11 @@ tc.research_text.FOCUSPOUCH=Angenehmer Behälter für deine Foki
 tc.research_page.FOCUSPOUCH.1=Nachdem du eine ansehnliche Sammlung an Foki zusammengetragen hast, ist es recht lästig, sie alle immer mit dir zu tragen. Diese Tasche löst das Problem.<BR>Immer, wenn du den Fokus an deinem Zauberstab wechselst, versucht der Stab zunächst, den Fokus in der Tasche abzulegen. Ebenso kannst du mit der Tasche rechtsklicken, um Foki manuell abzulegen.<BR>Er kann ebenfalls in deinem Gürtel-Slot getragen werden.
 #
 tc.research_name.BOOTSTRAVELLER=Stiefel des Reisenden
-tc.research_text.BOOTSTRAVELLER=These boots were made for walking
+tc.research_text.BOOTSTRAVELLER=Diese Stiefel wurden für das Laufen erschaffen
```
Comment (Vexatos) [inline de_DE.lang]: Do not translate references like this.

File: de_DE.lang [hunk 5]
```diff
@@ -1305,7 +1305,7 @@ tc.research_name.COREUSE=Golemkern: Nutzen
...
 tc.research_page.COREUSE.2=Wenn der Golem nicht an ein Inventar gebunden ist, wird er versuchen, seine Aktion mit leeren Händen auszuführen - einfach auf das Ziel zu klicken, wie angeordnet.<LINE>Du bist fähig, dem Golem zu sagen, wie auf etwas 'geklickt' werden soll - statt nur zu sagen, rechts- oder linkszuklicken, kannst du auch bestimmen, ob der Golem bei der Ausführung schleichen soll oder nicht.<BR>Du kannst auch bestimmen, ob er nur auf Luft statt auf einen Block klicken soll.
-tc.research_page.COREUSE.3=      §nLuft-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_air.png:0:0:255:255:0.0625</IMG>Dieses Upgrade erhöht die Geschwindigkeit, mit welcher der Golem den Zielblock oder -raum klicken kann.
```
Comment (Vexatos) [inline de_DE.lang]: See https://github.com/Azanor/thaumcraft/pull/1696/files#r26854445

File: de_DE.lang [hunk 6]
```diff
@@ -1332,19 +1332,19 @@ tc.research_page.COREBUTCHER.1=Dieser Kern ist eine Verbesserung des normalen Sc
...
-tc.research_page.CORELIQUID.2=    §nEntropie-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_entropy.png:0:0:255:255:0.0625</IMG>Mit diesem Upgrade wird der Golem versuchen, den am weitesten entfernten Flüssigkeitsblock zuerst aufzunehmen, der an einen Block, den du markiert hast, anschließt. Dies erlaubt ihnen, ganze Becken einer Flüssigkeit zu leeren, nicht nur den Block, den du markiert hast.<BR>Er wird lediglich so weit suchen, wie er blicken kann; die Distanz wird als Anzahl der Blöcke vom markierten zum Zielblock gemessen, nicht in einer geraden Linie.
+tc.research_page.CORELIQUID.2= §nEntropie-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_entropy.png:0:0:255:255:0.0625</IMG>Mit diesem Upgrade wird der Golem versuchen, den am weitesten entfernten Flüssigkeitsblock zuerst aufzunehmen, der an einen Block, den du markiert hast, anschließt. Dies erlaubt ihnen, ganze Becken einer Flüssigkeit zu leeren, nicht nur den Block, den du markiert hast.<BR>Er wird lediglich so weit suchen, wie er blicken kann; die Distanz wird als Anzahl der Blöcke vom markierten zum Zielblock gemessen, nicht in einer geraden Linie.
```
Comment (Vexatos) [inline de_DE.lang]: https://github.com/Azanor/thaumcraft/pull/1696/files#r26854445

File: de_DE.lang [hunk 7]
```diff
@@ -1314,7 +1314,7 @@ tc.research_page.COREFILL.1=Mit diesem Kern ausgestattete Golems durchsuchen von
 tc.research_name.COREHARVEST=Golemkern: Ernten
 tc.research_text.COREHARVEST=Animierter Farmer
 tc.research_page.COREHARVEST.1=Golems mit diesem Kern muss kein Inventar zugewiesen werden. Sie suchen reifes Getreide, Zuckerrohr und andere angebaute Pflanzen in einem kleinen Umkreis und ernten sie. Die Ernte wird auf den Boden fallen gelassen und muss irgendwie gesammelt werden.<BR>Leider sind sie nicht intelligent genug, um geerntetes Getreide nachzupflanzen, aber du bist dir sicher, dass du diesen Makel mit etwas Aufwand beheben könntest.
-tc.research_page.COREHARVEST.2=     §nOrdnungs-Upgrade freigeschaltet§r <IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Ernte-Golems mit diesem Upgrade versuchen nun, geerntetes Getreide neu zu pflanzen. Dies kann fehlschlagen, wenn die Pflanze keinen brauchbaren Samen hat fallen lassen.
+tc.research_page.COREHARVEST.2= §nOrdnungs-Upgrade freigeschaltet§r <IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Ernte-Golems mit diesem Upgrade versuchen nun, geerntetes Getreide neu zu pflanzen. Dies kann fehlschlagen, wenn die Pflanze keinen brauchbaren Samen hat fallen lassen.
```
Comment (Vexatos) [inline de_DE.lang]: Same here

File: de_DE.lang [hunk 8]
```diff
@@ -1323,7 +1323,7 @@ tc.research_page.CORELUMBER.1=Wie Ernte-Golems können auch Holzfäller-Golems 
 tc.research_name.COREGUARD=Golemkern: Schützen
 tc.research_text.COREGUARD=Schützt deine Wertsachen
 tc.research_page.COREGUARD.1=Golems mit diesem Kern arbeiten als Wachen, sie greifen jede feindliche Kreatur in Reichweite an.<BR>Du bist sicher, dass du mit ein wenig Bastelei die Golems präziser bei der Auswahl ihrer Ziele machen kannst, aber im Moment suchen sie nur aggressive Monster, außer Creeper.
-tc.research_page.COREGUARD.2=     §nOrdnungs-Upgrade freigeschaltet§r <IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Mit diesem Upgrade kannst du festlegen, welche Kreaturen der Golem angreifen soll. Außerdem kann der Golem auch andere Wesen als Feinde angreifen.
```
Comment (Vexatos) [inline de_DE.lang]: Same here

File: de_DE.lang [hunk 9]
```diff
@@ -1332,19 +1332,19 @@ tc.research_page.COREBUTCHER.1=Dieser Kern ist eine Verbesserung des normalen Sc
...
 tc.research_page.COREFISHING.1=Dieser Kern ermöglicht es einem Golem, für dich zu angeln.<BR>Für beste Ergebnisse muss der Golem nahe tiefen und offenen Wassers platziert werden.<BR>Die Stärke des Golems bestimmt die Angelgeschwindigkeit.<BR>
-tc.research_page.COREFISHING.2=      §nLuft-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_air.png:0:0:255:255:0.0625</IMG>Ein Angel-Golem mit diesem Upgrade hat eine geringe Chance, mehr als ein Ding auf einmal zu fangen.
-tc.research_page.COREFISHING.3=      §nFeuer-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_fire.png:0:0:255:255:0.0625</IMG>Ein Angel-Golem mit diesem Upgrade wird automatisch allen Fisch, den er fängt, kochen.
-tc.research_page.COREFISHING.4=     §nOrdnungs-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Odnungs-Upgrades werden die Chance für gute Angel-Resultate erhöhen.
-tc.research_page.COREFISHING.5=    §nEntropie-Upgrade freigeschaltet§r<IMG>thaumcraft:textures/items/golem_upgrade_entropy.png:0:0:255:255:0.0625</IMG>Entropie-Upgrades werden die Chance für schlechte Angel-Resultate verringern.
```
Comment (Vexatos) [inline de_DE.lang]: See https://github.com/Azanor/thaumcraft/pull/1696/files#r26854445

File: de_DE.lang [hunk 10]
```diff
@@ -1369,11 +1369,11 @@ tc.research_page.UPGRADEWATER.1=Dieses Upgrade erhöht die Sichtweite der meiste
...
 #
 tc.research_name.UPGRADEENTROPY=Golemupgrade: Entropie
 tc.research_text.UPGRADEENTROPY=Macht deinen Golem bissig... und ungenau
-tc.research_page.UPGRADEENTROPY.1=Golems mit diesem Upgrade verletzen automatisch alles, was ihnen Schaden zufügt.<BR>Ebenfalls werden Golems mit diesem Upgrade mehr Kontrolle darüber haben, welche Items sie suchen. Du wirst fähig sein, sie Metadaten un NBT-Daten ignorieren sowie im OreDictionary nach ähnlichen Items suchen zu lassen.<BR>Bestimme Golemkerne können von diesem Upgrade auch noch auf andere Weise profitieren, siehe also nach, ob zu ihnen weitere Seiten im Thaumonomikon hinzugefügt wurden.
+tc.research_page.UPGRADEENTROPY.1=Golems mit diesem Upgrade verletzen automatisch alles, was ihnen Schaden zufügt.<BR>Ebenfalls werden Golems mit diesem Upgrade mehr Kontrolle darüber haben, welche Items sie suchen. Du wirst fähig sein, sie Metadaten und NBT-Daten ignorieren sowie im OreDictionary nach ähnlichen Items suchen zu lassen.<BR>Bestimme Golemkerne können von diesem Upgrade auch noch auf andere Weise profitieren, siehe also nach, ob zu ihnen weitere Seiten im Thaumonomikon hinzugefügt wurden.
```
Comment (Vexatos) [inline de_DE.lang]: Could you change the "Bestimme" here to "Bestimmte" as well?

File: de_DE.lang [hunk 11]
```diff
@@ -1475,10 +1475,10 @@ tc.research_name.MIRROR=Spiegelmagie
...
-tc.research_page.MIRROR.3=Der transport von Items durch Spiegel strengt das magische Glas stark an, daher verlangsamt der Spiegel, um sich selbst vor dem Zersplittern zu bewahren, automatisch die Rate, mit welcher Items auf der anderen Seite ausgeworfen werden. Je höher die Instabilität, desto stärker wird die Transferrate vermindert.<BR>Diese Instabilität wird mit der Zeit weniger, aber kann auch durch direktes zuführen von §7Ordo§0 in den Spiegel negiert werden.
+tc.research_page.MIRROR.3=Der Transport von Items durch Spiegel strengt das magische Glas stark an, daher verlangsamt der Spiegel, um sich selbst vor dem Zersplittern zu bewahren, automatisch die Rate, mit welcher Items auf der anderen Seite ausgeworfen werden. Je höher die Instabilität, desto stärker wird die Transferrate vermindert.<BR>Diese Instabilität wird mit der Zeit weniger, aber kann auch durch direktes zuführen von §7Ordo§0 in den Spiegel negiert werden.
 #
 tc.research_name.MIRRORHAND=Magischer Handspiegel
-tc.research_text.MIRRORHAND=Persönliche Reflektion
+tc.research_text.MIRRORHAND=Persönliche Reflexion
```
Comment (Vexatos) [inline de_DE.lang]: Why the heck did I do this... Thanks anyway.

File: de_DE.lang [hunk 12]
```diff
@@ -1522,7 +1522,7 @@ tc.research_page.INFUSIONENCHANTMENT.3=Zusätzliche Kosten entstehen, wenn das I
 tc.research_name.ARCANELAMP=Arkane Lampe
 tc.research_text.ARCANELAMP=Große Erleuchtung
 tc.research_page.ARCANELAMP.1=Diese mystische Lampe kann an Wänden, Böden oder Decken angebracht werden und spendet helles und gleichmäßiges Licht.<BR>Zusätzliche Lichtquellen können bis zu 16 Blöcke entfernt entstehen, wenn der Lichtlevel dort unterhalb von es "sicheren" Levels liegt. Vergiss niemals: Nur DU kannst die Zombie-Apokalypse verhindern.<BR>Diese sekundären Lichtquellen verschwinden, kurz nachdem die Lampe entfernt wurde.<LINE>Du bist sicher, dass du noch weitere Verwendung für diese Lampe finden wirst, also schlage nochmal nach, wenn du weitere Entdeckungen gemacht hast.
-tc.research_page.ARCANELAMP.2=           §nArkaner Bohrer§r<BR>In Verbindung mit dem Gestell des Arkanen Bohrers erhält die Arkane Lampe eine weitere Fähigkeit: Sie beleuchtet den Tunnel, den der Bohrer gräbt.
```
Comment (Vexatos) [inline de_DE.lang]: See https://github.com/Azanor/thaumcraft/pull/1696/files#r26854445

File: de_DE.lang [hunk 13]
```diff
@@ -1545,12 +1545,12 @@ tc.research_name.TRAVELTRUNK=Reisekoffer
...
-tc.research_page.TRAVELTRUNK.UAI=      §nGolemupgrade: Luft§r<BR>Mit diesem Upgrade ist der Reisekoffer in der Lage, sich schneller zu bewegen.
-tc.research_page.TRAVELTRUNK.UEA=      §nGolemupgrade: Erde§r<BR>Mit diesem Upgrade kann der Reisekoffer mehr Items transportieren und erhält eine zusätzliche Reihe Inventarplatz.
-tc.research_page.TRAVELTRUNK.UFI=     §nGolemupgrade: Feuer§r<BR>Mit diesem Upgrade verteidigt der Reisekoffer seinen Meister, wenn dieser angegriffen wird.
-tc.research_page.TRAVELTRUNK.UWA=     §nGolemupgrade: Wasser§r<BR>Mit diesem Upgrade kann der Reisekoffer nur von seinem Meister aufgenommen oder geöffnet werden.<BR>Zusätzlich wird er selbst gegen die größten Schäden sehr widerstandsfähig.
-tc.research_page.TRAVELTRUNK.UOR=    §nGolemupgrade: Ordnung§r<BR>Mit diesem Upgrade wird der Reisekoffer seinen Inhalt behalten, wenn er mit der Glocke des Golemanten aufgenommen wurde. Er lässt seinen Inhalt fallen, wenn shift-klick benutzt wird.
-tc.research_page.TRAVELTRUNK.UEN=    §nGolemupgrade: Entropie§r<BR>Mit diesem Upgrade wird der Reisekoffer automatisch alle Items in der Nähe aufsaugen, für die er Platz hat.
```
Comment (Vexatos) [inline de_DE.lang]: See https://github.com/Azanor/thaumcraft/pull/1696/files#r26854445

File: de_DE.lang [hunk 14]
```diff
@@ -1699,8 +1699,8 @@ tc.research_page.ELDRITCHMAJOR.2=<IMG>thaumcraft:textures/misc/eldritchajor2.png
...
+tc.research_page.VOIDMETAL.2=wenn hart getroffen, wird es so hart wie Thaumium, manchmal sogar härter. Noch beeindruckenderweise repariert das Metall sich nach allem außer dem allerverheerendsten Schaden von selbst, da dass Metall langsam in seine Ursprungsform zurückfließt.<BR>Der einzige Nachteil ist, dass es nicht so leicht zu verzaubern ist wie Thaumium, jedoch ist es immer noch besser als gewöhnliche Materialien wie Eisen.<BR>Du hast entdeckt, dass Vorsicht beim Umgang mit diesem Metall geboten ist - von ihm verletzt zu werden, scheint temporär deine Stärke auszusaugen.
```
Comment (Vexatos) [inline de_DE.lang]: Needs to be `beeindruckenderweise`

File: de_DE.lang [hunk 15]
```diff
@@ -1789,5 +1789,5 @@ tc.research_page.SINSTONE.1=Du hast eine interessante Entdeckung während deiner
...
+tc.research_page.PRIMALCRUSHER.2=ein Werkzeug unerreichter zerstörerischer Fähigkeiten erschaffen, dass die meisten Eigenschaften seiner Komponenten beibehalten hat.<BR>Der Primärhäcksler kann mit Leichtigkeit gleichsam durch Erde und Stein graben, und das in einem 3x3-Gebiet. Wie die Spitzhacke des Kerns gräbt er manchmal Native Klumpen statt rohen Erzes aus.<BR>Er repariert sich langsam von selbst, ist sehr leicht zu verzaubern und fungiert zur Not auch als relativ gute Waffe.<BR>Leider scheint er rein destruktiver Natur zu sein; er hat alle Konstruktions- und Erkennungsfähigkeiten seiner Komponenten verloren.
```
Comment (Vexatos) [inline de_DE.lang]: I actually like the old German word "Erd" here... Don't you?

**Comment by Vexatos** [comment]
The rest looks decent, thank you a lot for looking through all this text!

File: de_DE.lang (hunk 14 above)
Comment (noname42) [inline de_DE.lang]: never heard "beeindruckendererweise" and i'm native german

File: de_DE.lang (hunk 3 above)
Comment (noname42) [inline de_DE.lang]: This is because I copied everyting in a text editor and than copied it back.

File: de_DE.lang (hunk 4 above)
Comment (noname42) [inline de_DE.lang]: I didn't know this is a reference o.o

File: de_DE.lang (hunk 2 above)
Comment (noname42) [inline de_DE.lang]: I'm not 100% sure but i think the "," is optional here

File: de_DE.lang (hunk 1 above)
Comment (noname42) [inline de_DE.lang]: I changed it because "virus" is more understandable

File: de_DE.lang (hunk 4 above)
Comment (noname42) [inline de_DE.lang]: It's actually "These Boots Are Made for Walkin’"

Comment (noname42) [inline de_DE.lang]: same goes for "Wizards staves have knobs on the end" assume?

File: de_DE.lang (hunk 1 above)
Comment (Vexatos) [inline de_DE.lang]: Fair enough. I like it.

File: de_DE.lang (hunk 2 above)
Comment (Vexatos) [inline de_DE.lang]: Yes, but I used commas wherever possible to keep the text easier to read, so please stay consistent.

File: de_DE.lang (hunk 14 above)
Comment (Vexatos) [inline de_DE.lang]: Nevermind, you're correct on this one.

File: de_DE.lang (hunk 3 above)
Comment (Vexatos) [inline de_DE.lang]: You'll still need to change this.

File: de_DE.lang [hunk 16]
```diff
@@ -918,18 +918,18 @@ tc.research_page.NODES.3=Dadurch kann er verschwinden oder der Integrität der N
...
-tc.research_page.PLANTS.4=§l        §nAschperle§r<BR><IMG>thaumcraft:textures/misc/research5.png:0:0:128:128:.6</IMG><BR>Lediglich in den heißesten Gegenden der Welt wächst diese Pflanze, genährt von elementarem Feuer. Wird sie geerntet und zermahlen, erhält man ein feines Pulver, das Lohenstaub ähnelt.
-tc.research_page.PLANTS.5=§l        §nManaschote§r<BR><IMG>thaumcraft:textures/misc/research5.png:0:128:128:128:.6</IMG><BR><BR><BR>Diese seltsame, schotenartige Pflanze kann nur an Bäumen in magischen Wäldern gefunden werden. Sie gedeihen nur sehr langsam und geben bei der Ernte selten mehr als eine oder zwei Bohnen.
+tc.research_page.PLANTS.1=§l §nGroßholzbäume§r<BR><IMG>thaumcraft:textures/misc/research4.png:128:128:128:128:.6</IMG><BR>Die großen und majestätischen Bäume können überall vereinzelt gefunden werden.<BR>Ihr Holz ist in der Lage, eine beeindruckende Menge Magie aufzunehmen, was es zu einem beliebten Material für magische Geräte macht.
```
Comment (Vexatos) [inline de_DE.lang]: I'm sorry, but you will need to fix all of these.

File: de_DE.lang (hunk 8 above)
Comment (Vexatos) [inline de_DE.lang]: This one apparently got one too few.

File: de_DE.lang [hunk 17]
```diff
@@ -1314,7 +1314,7 @@ tc.research_page.COREFILL.1=Mit diesem Kern ausgestattete Golems durchsuchen von
 tc.research_name.COREHARVEST=Golemkern: Ernten
 tc.research_text.COREHARVEST=Animierter Farmer
 tc.research_page.COREHARVEST.1=Golems mit diesem Kern muss kein Inventar zugewiesen werden. Sie suchen reifes Getreide, Zuckerrohr und andere angebaute Pflanzen in einem kleinen Umkreis und ernten sie. Die Ernte wird auf den Boden fallen gelassen und muss irgendwie gesammelt werden.<BR>Leider sind sie nicht intelligent genug, um geerntetes Getreide nachzupflanzen, aber du bist dir sicher, dass du diesen Makel mit etwas Aufwand beheben könntest.
-tc.research_page.COREHARVEST.2=     §nOrdnungs-Upgrade freigeschaltet§r <IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Ernte-Golems mit diesem Upgrade versuchen nun, geerntetes Getreide neu zu pflanzen. Dies kann fehlschlagen, wenn die Pflanze keinen brauchbaren Samen hat fallen lassen.
+tc.research_page.COREHARVEST.2=    §nOrdnungs-Upgrade freigeschaltet§r <IMG>thaumcraft:textures/items/golem_upgrade_order.png:0:0:255:255:0.0625</IMG>Ernte-Golems mit diesem Upgrade versuchen nun, geerntetes Getreide neu zu pflanzen. Dies kann fehlschlagen, wenn die Pflanze keinen brauchbaren Samen hat fallen lassen.
```
Comment (Vexatos) [inline de_DE.lang]: This one apparently got too few spaces.

File: de_DE.lang [hunk 18]
```diff
@@ -1361,19 +1361,19 @@ tc.research_page.UPGRADEEARTH.1=Dieses Upgrade erhöht die Anzahl an Items, die
...
 #
 tc.research_name.UPGRADEENTROPY=Golemupgrade: Entropie
 tc.research_text.UPGRADEENTROPY=Macht deinen Golem bissig... und ungenau
-tc.research_page.UPGRADEENTROPY.1=Golems mit diesem Upgrade verletzen automatisch alles, was ihnen Schaden zufügt.<BR>Ebenfalls werden Golems mit diesem Upgrade mehr Kontrolle darüber haben, welche Items sie suchen. Du wirst fähig sein, sie Metadaten un NBT-Daten ignorieren sowie im OreDictionary nach ähnlichen Items suchen zu lassen.<BR>Bestimme Golemkerne können von diesem Upgrade auch noch auf andere Weise profitieren, siehe also nach, ob zu ihnen weitere Seiten im Thaumonomikon hinzugefügt wurden.
+tc.research_page.UPGRADEENTROPY.1=Golems mit diesem Upgrade verletzen automatisch alles, was ihnen Schaden zufügt.<BR>Ebenfalls werden Golems mit diesem Upgrade mehr Kontrolle darüber haben, welche Items sie suchen. Du wirst fähig sein, sie Metadaten und NBT-Daten ignorieren sowie im OreDictionary nach ähnlichen Items suchen zu lassen.<BR>Bestimmte Golemkerne können von diesem Upgrade auch noch auf andere Weise profitieren, siehe also nach, ob zu ihnen weitere Seiten im Thaumonomikon hinzugefügt wurden.
```
Comment (Vexatos) [inline de_DE.lang]: Thanks for fixing all of these!

File: de_DE.lang (hunk 15 above)
Comment (Vexatos) [inline de_DE.lang]: I would still change this.

**Comment by Vexatos** [comment]
:+1: Thank you for fixing everything
@Azanor, this is fine to merge now.

**Pull request merged**